### PR TITLE
End to end testing functionality (MVP)

### PR DESCRIPTION
**Pull request by ThomasGalesloot**
Initially planned functionality for end-to-end testing has been completed (MVP). There are future features that represent significant improvements, such as shared inputs and parameters not present in the end-to-end testing functionality yet.

### CODE REVIEW

**Review by KeeganShaw-GIS — COMMENTED** [review]
Nice! I would say automating the initialization would be useful. So user specifies which E2E they want, and the parameters are already set up for them and ready to go.  No need to copy or delete files by hand

File: LDAR_Sim/src/default_parameters/g_default.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-version: '2.0'
-parameter_level: 'global'
-input_directory: './inputs'
-output_directory: './outputs'
-n_processes:  '_placeholder_int_'
+version: "2.0"
+parameter_level: "global"
+input_directory: "./inputs"
+output_directory: "./outputs"
+n_processes: 5
```
[the remark this answers is not in the document]
Comment (ThomasGalesloot) [reply LDAR_Sim/src/default_parameters/g_default.yml]: Done

File: LDAR_Sim/testing/end_to_end_testing/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    pregenerate_leaks: True
+    ```
+
+4. Change the global parameter "n_simulations" to 1
+
+    ```yaml
+    n_simulations: 1
+    ```
+
+5. Delete the generator folder from the inputs folder if it exists
```
[the remark this answers is not in the document]
Comment (ThomasGalesloot) [reply LDAR_Sim/testing/end_to_end_testing/README.md]: Done

File: LDAR_Sim/testing/end_to_end_testing/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+6. Copy the inputs folder and the folder containing the simulation parameters inside the test_case_creator folder
```
[the remark this answers is not in the document]
Comment (ThomasGalesloot) [reply LDAR_Sim/testing/end_to_end_testing/README.md]: Done

**Review by SEAJang — APPROVED** [review]
Looks great. Amazing work!

**Pull request merged**